docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    def calculate_paf_and_outcome_bias_dichotomous_risks(r1_exp, r1_rr, r2_exp, r2_rr, rho,
</pre><p>Can we start with dichotomous, as in the other scenarios?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>